### PR TITLE
Pset_BuiltSystemTypeWaterproofing

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_BuiltSystemTypeWaterproofing

## Description: 

Properties for waterproofing systems

| Property                     | Datatype                  | Description                                 |
| ---------------------------- | ------------------------- | ------------------------------------------- |
| WaterproofingSystem          | PEnum_WaterproofingSystem | Indicates the type of waterproofing system  |
| WatertightnessClassification | IfcLabel                  | Specifies the watertightness classification |

## Enumeration: PEnum_WaterproofingSystem

| Constant   | Description |
| ---------- | ----------- |
| WATERTIGHT |             |
| DRAINED    |             |

### CODE REVIEW

**Comment by SergejMuhic** [comment]
No descriptions for constants.

**Comment by JontyDunn** [comment]
WATERTIGHT The groundwater water is prevented from penetrating the tunnel. The full water pressure acts on the tunnel lining

DRAINED  The groundwater flows freely through a drainage layer into the tunnel and is collected with a seepage pipe.  A reduced water pressure acts on the tunnel lining

File: IFC4x3/Properties/w/WatertightnessClassification_3o5MfIEcL6VubC3CVWfHTs/DocProperty.xml
Comment (SergejMuhic) [inline IFC4x3/Properties/w/WatertightnessClassification_3o5MfIEcL6VubC3CVWfHTs/DocProperty.xml:1]: Classification references should be modeled with `IfcClassificationReference`.

File: IFC4x3/PropertyEnumerations/w/PEnum_WaterproofingSystem.xml
Comment (SergejMuhic) [inline IFC4x3/PropertyEnumerations/w/PEnum_WaterproofingSystem.xml:1]: Would be nice to have a short description for the enum.

**Review by SergejMuhic — CHANGES_REQUESTED** [review]

**Comment by SergejMuhic** [comment]
@michel-rives add NOTKNOWN?

**Comment by Michel-Rives** [comment]
> @Michel-Rives add NOTKNOWN?

Yes

**Comment by SergejMuhic** [comment]
> > @Michel-Rives add NOTKNOWN?
> 
> Yes

Added.

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**